src/components/chart/chart.js

```diff
@@ -178,7 +208,13 @@ class HighchartsBaseChart {
                 this.useStackedLayout,
                 this.extraLines,
             );
-            mobileChartOptions = this.mergeConfigs(mobileChartOptions, mobileColumnChartOptions);
+            const mobileBoxplotChartOptions = this.columnChart.getColumnChartMobileOptions(
```

Comment:
fyi This is where the `getPointPadding` frpm the column chart is effectively being pulled in.

Reply:
Discussed with Precious and this one is for getting the mobile point padding so needs to remain. It gets called at desktop too, but we're getting the mobile values. Probably would need to refactor how this works in order to change it, so leaving for now.